src/test/java/org/folio/edge/sip2/repositories/CirculationRepositoryTests.java

```diff
@@ -483,6 +524,22 @@ void cannotCheckout(String errorMessage, List<String> expectedErrors, Vertx vert
         .cancel(FALSE)
         .build();
 
+    final JsonObject response = new JsonObject()
+      .put("instances", new JsonArray()
+        .add(new JsonObject()
+          .put("id", "7fbd5d84-62d1-44c6-9c45-6cb173998bbd")
+          .put("title","Bridget Jones's Baby: the diaries")
+          .put("contributors", new JsonArray().add(new JsonObject().put("name","Fielding, Helen")))))
+      .put("totalRecords", 1);
+
+    final String expectedPath = "/search/instances?limit=1&query=%28items.barcode%3D%3D453987605438%29";
+
+    when(mockFolioProvider.retrieveResource(any()))
+      .thenReturn(Future.succeededFuture(new FolioResource(response,
+        MultiMap.caseInsensitiveMultiMap().add("x-okapi-token", "1234"))));
+
+
+
```

Comment:
Please remove additional returns and just have one line between statements

Reply:
Added please check

Reply:
Still seeing more than one newline here. There should only be one between the two `when` statements.